recipes/ccdproc/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "ccdproc" %}
+{% set version = "1.2.0" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d96fcb6c45c2d060ef915380ed75d230693befa3f9427fdeed0ec8463abfac38" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+  fn: '{{ name }}-{{ version }}.tar.gz'
```

Comment:
BTW we can drop the `fn` now. It has been a while that `conda-build` can derive it from the `URL`.